src/Intervention/Image/Gd/Encoder.php

```diff
@@ -121,4 +121,16 @@ protected function processPsd()
             "PSD format is not supported by Gd Driver."
         );
     }
+
+    /**
+     * Processes and returns encoded image as AVIF string
+     *
+     * @return string
+     */
+    protected function processAvif()
+    {
+        throw new NotSupportedException(
```

Comment:
Actually, GD supports avif: https://github.com/libgd/libgd/releases/tag/gd-2.3.2

Reply:
@freshleafmedia could you update this file to include the GD support?

```php
    protected function processAvif()
    {
        if ( ! function_exists('imageavif')) {
            throw new NotSupportedException(
                "Avif format is not supported by PHP installation."
            );
        }

        ob_start();
        imageavif($this->image->getCore(), null, $this->quality);
        $this->image->mime = defined('IMAGETYPE_AVIF') ? image_type_to_mime_type(IMAGETYPE_AVIF) : 'image/avif';
        $buffer = ob_get_contents();
        ob_end_clean();

        return $buffer;
    }
```

Reply:
GD Support is not yet available in PHP GD.

https://bugs.php.net/bug.php?id=80828

Reply:
That's not quite accurate, https://github.com/php/php-src/pull/7026 is merged so GD with avif will be available in the next patch version (8.1)

Reply:
You're both right. AVIF is now supported in libgd. The support in PHP is now merged, and it's set to be released in PHP 8.1. ☺️